File: app/src/main/java/com/gutotech/resultadosloterias/api/RetrofitConfig.java
package com.gutotech.resultadosloterias.api;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitConfig {
    private Retrofit retrofit;

    public RetrofitConfig() {
        retrofit = new Retrofit.Builder()
                .baseUrl(ResultadosLoteriasService.baseUrl)
                .addConverterFactory(GsonConverterFactory.create())
                .build();
    }

    public ResultadosLoteriasService getResultadosLoteriasService() {
        return retrofit.create(ResultadosLoteriasService.class);
    }
}

File: app/src/main/java/com/gutotech/loteria/data/retrofit/RetrofitConfig.kt
package com.gutotech.loteria.data.retrofit

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

class RetrofitConfig {

    private val retrofit = Retrofit.Builder()
        .baseUrl("https://apiloterias.com.br/api0/")
        .addConverterFactory(GsonConverterFactory.create())
        .build()

    fun getResult() = retrofit.create(LotteryService::class.java)

}